src/OpenTelemetry.Instrumentation.ConfluentKafka/InstrumentedProducer.cs

```diff
@@ -329,7 +326,7 @@ private static void RecordPublish(TopicPartition topicPartition, TimeSpan durati
 
     private Activity? StartPublishActivity(DateTimeOffset start, string topic, Message<TKey, TValue> message, int? partition = null)
     {
-        var spanName = string.Concat(topic, " ", PublishOperationName);
+        var spanName = string.Concat(topic, " ", ConfluentKafkaCommon.PublishOperationName);
         var activity = ConfluentKafkaCommon.ActivitySource.StartActivity(name: spanName, kind: ActivityKind.Producer, startTime: start);
```

Comment:
unrelated to this PR, but why would you ever need to pass start time to publish activity?

Reply:
thanks for the explanation!

I don't think we try to be so precise in other places, but it definitely makes sense.